README.md

```diff
@@ -44,42 +44,47 @@ git submodule update --init --recursive
 
 ## Contributing
 
-Contributions are welcome! 
+Contributions are welcome!
 
-Learn more about [code](CONTRIBUTING.md#contributing-code), [data](CONTRIBUTING.md#contributing-data), and [documentation](CONTRIBUTING.md#contributing-documentation) contributions. 
-We also have a [mailing list](https://groups.google.com/g/osv-discuss). 
+Learn more about [code](CONTRIBUTING.md#contributing-code), [data](CONTRIBUTING.md#contributing-data), and [documentation](CONTRIBUTING.md#contributing-documentation) contributions.
+We also have a [mailing list](https://groups.google.com/g/osv-discuss).
 
-Do you have a question or a suggestion? Please [open an issue](https://github.com/google/osv.dev/issues). 
+Do you have a question or a suggestion? Please [open an issue](https://github.com/google/osv.dev/issues).
 
 ## Third party tools and integrations
 
 There are also community tools that use OSV. Note that these are community built
-tools and unsupported by the core OSV maintainers.
-
--   [Betterscan.io: Code Scanning/SAST/Static Analysis/Linting using many
-    tools/Scanners with One Report (Code,
-    IaC)](https://github.com/marcinguy/betterscan-ce)
--   [bomber](https://github.com/devops-kung-fu/bomber)
--   [Cortex XSOAR](https://github.com/demisto/content)
--   [dependency-management-data](https://dmd.tanna.dev)
--   [Dependency-Track](https://github.com/DependencyTrack/dependency-track)
--   [dep-scan](https://github.com/AppThreat/dep-scan)
--   [G-Rath/osv-detector](https://github.com/G-Rath/osv-detector): A scanner
-    that uses the OSV database.
--   [GUAC](https://guac.sh)
--   [it-depends](https://github.com/trailofbits/it-depends)
--   [.NET client library and support for the schema](https://github.com/JamieMagee/osv.net)
--   [OSS Review Toolkit](https://github.com/oss-review-toolkit/ort)
--   [OSV4k: a Java/Kotlin MPP library for serialization and deserialization of OSV schema](https://github.com/saveourtool/osv4k)
--   [Packj](https://github.com/ossillate-inc/packj)
--   [pip-audit](https://pypi.org/project/pip-audit/)
--   [Renovate](https://github.com/renovatebot/renovate)
--   [rosv: an R package to access the OSV database and help administer Posit Package Manager](https://github.com/al-obrien/rosv)
--   [Rust client library](https://github.com/gcmurphy/osv)
--   [Skjold: Security audit python project dependencies against several security
-    advisory databases](https://github.com/twu/skjold)
--   [Trivy](https://github.com/aquasecurity/trivy)
--   [IronDome: SCA scanner for Ruby applications](https://rubygems.org/gems/iron_dome)
--   [OSV module in x-cmd: A shell CLI for OSV.dev API with osv-scanner integration](https://x-cmd.com/mod/osv)
-
-Feel free to send a PR to add your project here.
+tools and unsupported and not endorsed by the core OSV maintainers. You may wish
```

Comment:
Hmm the wording here seems a bit weird with the two ands.

Reply:
> Hmm the wording here seems a bit weird with the two ands.

I agree on a second look, thanks. Fixed in 66ac9485d5b0bf9e81c8f3421df1daffae3001f6